src/main/webapp/app/exercises/shared/exercise/exercise-lti-configuration.component.html

```diff
@@ -9,55 +9,75 @@
             LTI Configuration for exercise {{ exercise?.title }}
```

Comment:
While you're at it, all strings (mainly input labels) in this file should be moved to translations.

Reply:
Thanks for the help with the translations, I already fixed them. Actually I considered moving the input labels to the translations but I don't think it makes sense since we're following the LTI standard which is fully in English and the strings are keywords in the standard. What do you think? I can still move them to the translations and just keep the english version even for german.

Reply:
I think you could still translate them, if the user wishes they can always switch the language in Artemis with two clicks. :)
Here is a German documentation about the LTI integration: https://online-uebungssystem.fernuni-hagen.de/download/LTI_Moodle/LTI_TechnicalDoc.html

Reply:
Thanks, I've moved all the strings to the translations (except for the values themselves which have to be in english) and tried translating them. If you could check the translations again I'd appreciate it